frontend/src/entry-forms/Transaction.svelte

```diff
@@ -1,16 +1,11 @@
-<script lang="ts" context="module">
```

Comment:
Why was this code removed? We still want to have this logic to be able to input tags and links

Reply:
I removed it because the linter told me that it wasn't used anymore. I will try to double check it again in the next few days. Same as for your other comment. Thanks for the hint with the svelte-check error. I probably wouldn't have found that one.

Reply:
So I did had a look at it again and I inserted it again. Tags and links will now be parsed properly before adding the new entry to the beancount file.